azure-pipelines-dev.yml

```diff
@@ -329,7 +360,6 @@ stages:
     jobs:
       - job: mysql
         condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.GEChanged'], true)
-        timeoutInMinutes: 120
```

Comment:
No longer necessary since we're cutting down test times

Reply:
Isn't this to force stop close run-away tests?
If so we should probably just reduce the `timeoutInMinutes` but not remove it.

Reply:
By default this is 60 I believe. Removing it doesn't actually set remove this constraint.